### PR TITLE
Topic/exceptions

### PR DESCRIPTION
**Pull request by caustin-usgs**
Making sure: 
1) Exceptions for critical functions result in the container exit with a non-zero status code
2) inability to connect to the API throws an exception

removing unused functions, ie handle-orders, queue-products, etc.  consolidating logging setup.
simplifying main() etc

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -36,7 +36,10 @@ pipeline {
                 script {
                     // Test commands inside of the built image
                     CUSTOM_IMAGE.inside {
-			sh: """nose2 --with-coverage""", label: "Running unit tests"
+			            sh """
+			                cd /src
+			                nose2 --with-coverage
+			            """
```
Comment (danzelenak-usgs) [inline Jenkinsfile]: This doesn't seem to work - the file structure is not as I'd expect inside the container..getting /src does not exist but it's created in the Dockerfile.

File: processing/main.py [hunk 2]
```diff
@@ -354,83 +131,75 @@ def work(cfg, params, developer_sleep_mode=False):
...
+        except Exception as e2:
+            logger.exception('Problem archiving log files. error: {}'.format(e2))
 
-    # export values for the container environment
-    config.export_environment_variables(cfg)
+        try:
+            logger.debug('Attempting to set product error, order_id: {}\nproduct_id: {}'.format(order_id, product_id))
+            logged_contents = EspaLogging.read_logger_file(settings.PROCESSING_LOGGER)
+            error_log = "Processing Log: {}\n\nException: {}".format(logged_contents, e)
+            server.set_scene_error(product_id, order_id, processing_location, str(e))
```
Comment (danzelenak-usgs) [inline processing/main.py]: Noted that we should pass error_log rather than the exception e to set_scene_error()

File: Jenkinsfile (hunk 1 above)
Comment (danzelenak-usgs) [reply Jenkinsfile]: Not critical for this PR - something I need to work on though.

Comment (caustin-usgs) [reply Jenkinsfile]: right. i started this with your branch, and didn't mess with the Jenkins setup at all.

**Pull request merged**